src/stories/5-custom/filter/filter-checkbox-item.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { CommonCustomStoryProps, FrontendEngine } from "../../common";
+import { IFilterSchema } from "../../../components/custom/filter/filter/types";
+
+export default {
+	title: "Custom/Filter/Filter-Checkbox",
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Filter Checkbox</Title>
+					<Description>Widget to select multiple options</Description>
+					<Heading>Props</Heading>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Examples" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...CommonCustomStoryProps("filter-checkbox"),
+		children: {
+			description: "Elements or string that is the descendant of this component",
+			table: {
+				type: {
+					summary: "TFrontendEngineFieldSchema | string | (string | TFrontendEngineFieldSchema)[]",
+				},
+			},
+			type: { name: "object", value: {}, required: true },
+		},
+		options: {
+			description: "A list of options that a user can choose from.",
+			table: {
+				type: {
+					summary: "{label: string, value: string}[]",
+				},
+			},
+			type: { name: "object", value: {} },
+		},
```

Comment:
- need argTypes for
  - collapsible
  - showDivider
  - showMobileDivider
  - validation
  - showIf